common/table.js

```diff
@@ -900,7 +900,12 @@
             if (vm.attributesAlreadyAttached) return;
             vm.attributesAlreadyAttached = true;
             vm.columnModels = [];
+
             vm.reference.columns.forEach(function (col) {
+                // if flag set, remove pseudo columns that are path columns
+                // NOTE: could check isPseudo, isPathColumn, and isForeignKey
+                if (vm.removePseudoColumns && col.isPseudo && col.isPathColumn) return;
```

Comment:
Is this the right way to check if a column should be ignored? If it is a pseudo column and a a path column?

Should it be `isPseudo && !isForeignKey`?